src/Microsoft.Extensions.FileProviders.Embedded/project.json

```diff
@@ -24,12 +24,15 @@
       "version": "1.1.0-*",
       "type": "build"
     },
-    "Microsoft.Extensions.FileProviders.Abstractions": "1.1.0-*"
+    "Microsoft.Extensions.FileProviders.Abstractions": "1.1.0-*",
+    "System.Runtime.Extensions": "4.1.0-*"
   },
   "frameworks": {
-    "netstandard1.0": {
+    "netstandard1.0": { },
+    "netstandard1.5": {
```

Comment:
Also we possibly want to cross compile to net451 to get this functionality on full framework.